py-polars/tests/unit/constructors/test_constructors.py

```diff
@@ -962,10 +962,20 @@ def test_init_pandas(monkeypatch: Any) -> None:
     assert df.rows() == [(1.0, 2.0), (3.0, 4.0)]
 
     # subclassed pandas object, with/without data & overrides
-    class XSeries(pd.Series):  # type: ignore[type-arg]
-        @property
-        def _constructor(self) -> type:
-            return XSeries
+    # Python 3.13 mypy will complain about unused type: ignore
+    if sys.version_info >= (3, 13):
+
+        class XSeries(pd.Series):
+            @property
+            def _constructor(self) -> type:
+                return XSeries
+
+    else:
+
+        class XSeries(pd.Series):  # type: ignore[type-arg]
+            @property
+            def _constructor(self) -> type:
+                return XSeries
```

Comment:
i think this is the new pandas-stubs release (which no longer supports python 3.9 😭 ) - might be worth addressing as a separate PR? could even just do `type: ignore[type-arg, unused-ignore]` and keep ti simple, it's not like it's a crucial class to type correctly 😄

Reply:
> which no longer supports python 3.9 😭

A few months. :)